docs/src/main/sphinx/release/release-406.md

```diff
@@ -0,0 +1,71 @@
+# Release 406 (13 Jan 2023)
+
+## General
+
+* Add support for [exchange spooling on HDFS](fte-exchange-hdfs) when
+  fault-tolerant execution is enabled. ({issue}`15160`)
+* Improve planner estimates for queries containing outer joins over a topN
+  operation. ({issue}`15428`)
+* Remove `legacy-phased` execution scheduler. ({issue}`15657`)
+* Fix failure when `WHERE` or `JOIN` clauses contain a `LIKE` expression with a
+  non-constant pattern or escape. ({issue}`15629`)
+* Fix inaccurate planner estimates for queries with filters on columns without
+  statistics. ({issue}`15642`)
+* Fix queries with outer joins failing when fault-tolerant execution is
+  enabled. ({issue}`15608`)
+* Fix potential query failure when using `MATCH_RECOGNIZE`. ({issue}`15461`)
+* Fix query failure when column masking or a row filter involves columns. ({issue}`15583`)
```

Comment:
The issue seem different from what this entry describes - it appears to be related to groups